### PR TITLE
Layer5 added as a istio partner

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

This pr is to add Layer5 as a partner of Istio

### CODE REVIEW

**Comment by ericvn** [comment]
This is the third PR I've reviewed asking to add Layer 5 to the partners page. 

This one appears to be missing the assets/images/layer5/layer5-tag-community-white-bg.png and I'm not sure why the Chinese change is added. I'll make this one run the test so one can verify that the actual graphic works, bit one of the other 2 PRs had the correct graphic link. This on at least has the location of the partner correct.

**Comment by frankbu** [comment]
This is a duplicate of #6184.

File: content/zh/docs/tasks/security/authentication/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Authentication
-description: Controlling mutual TLS and end-user authentication for mesh services.
+title: 认证
```
Comment (frankbu) [inline content/zh/docs/tasks/security/authentication/_index.md]: Please don't mix Chinese translation with other change in the same PR. Translation should be done in PRs with the `translation/chinese` label set.

**Pull request closed without merging**